lib/constants.ts

```diff
@@ -0,0 +1,3 @@
+const BASE_URL = process.env.PRODURL ?? 'http://localhost:3000';
```

Comment:
Where are we creating and getting our PRODURL from? Is it created happen after deployment and we have our code in production?

Reply:
You can specify environment variables in the command line, in a .env file, and/or with the host that we're hosting on (AWS).